app/controllers/facilities_controller.rb

```diff
@@ -1,46 +1,51 @@
 class FacilitiesController < ApplicationController
-    def index
-        @facilities = Facility.all
-    end
+  before_action :authenticate_user!, only: [:edit, :update, :destroy, :base_params, :create_params, :update_params]
```

Comment:
The parameter methods are already private, so you don't need to specify them in the before filter.

```suggestion
  before_action :authenticate_user!, only: [:edit, :update, :destroy]
```

Reply:
Made this correction in facilities, items and procedures controllers.